### PR TITLE
actions: Add mdformat, and set up action

### PR DESCRIPTION
**Pull request by ghost**
https://github.com/Open-Telecoms-Data/open-fibre-data-standard/issues/19

### CODE REVIEW

File: docs/reference/publication_formats/json.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ For guidance on paginating or streaming individual networks, see [how to publish
 
 The network package schema describes the structure of the container for publishing one or more networks in JSON format and for supporting pagination.
 
-For this version of OFDS, the canonical URL of the schema is [https://raw.githubusercontent.com/Open-Telecoms-Data/open-fibre-data-standard/0__1__0__beta/schema/network-package-schema.json](https://raw.githubusercontent.com/Open-Telecoms-Data/open-fibre-data-standard/0__1__0__beta/schema/network-package-schema.json). Use the canonical URL to make sure that your software, documentation or other resources refer to the specific version of the schema with which they were tested.
+For this version of OFDS, the canonical URL of the schema is [https://raw.githubusercontent.com/Open-Telecoms-Data/open-fibre-data-standard/0\_\_1\_\_0\_\_beta/schema/network-package-schema.json](https://raw.githubusercontent.com/Open-Telecoms-Data/open-fibre-data-standard/0__1__0__beta/schema/network-package-schema.json). Use the canonical URL to make sure that your software, documentation or other resources refer to the specific version of the schema with which they were tested.
```
Comment (ghost) [inline docs/reference/publication_formats/json.md:21]: I was surprised by escaping the underscore ... but it still seems to work!

**Comment by ghost** [comment]
No change log entry as none of the actual content should change - that ok?

File: docs/reference/schema.md [hunk 2]
```diff
@@ -638,12 +650,12 @@ Each `Identifier` has the following fields:
 
 Coordinates in all OFDS data must be specified in the coordinate reference system [required by GeoJSON](https://datatracker.ietf.org/doc/html/rfc7946#section-4):
 
-> The coordinate reference system for all GeoJSON coordinates is a geographic coordinate reference system, using the World Geodetic System 1984 (WGS 84) [WGS84] datum, with longitude and latitude units of decimal degrees.  This is equivalent to the coordinate reference system identified by the Open Geospatial Consortium (OGC) URN urn:ogc:def:crs:OGC::CRS84.
+> The coordinate reference system for all GeoJSON coordinates is a geographic coordinate reference system, using the World Geodetic System 1984 (WGS 84) \[WGS84\] datum, with longitude and latitude units of decimal degrees.  This is equivalent to the coordinate reference system identified by the Open Geospatial Consortium (OGC) URN urn:ogc:def:crs:OGC::CRS84.
```
Comment (ghost) [inline docs/reference/schema.md]: Surprised it's escaping this - output is same tho. However, I'm not sure what this is actually trying to do - is it meant to be a link?

Comment (duncandewhurst) [reply docs/reference/schema.md]: This was already fixed in the `github-actions` branch: https://github.com/Open-Telecoms-Data/open-fibre-data-standard/commit/1ea45986d4912b57315c9887846014e66cc5a2b2

How have the changes made in that branch been moved to this branch? I'm concerned that changes made in that branch are going to be lost.

File: manage.py [hunk 3]
```diff
@@ -777,6 +777,9 @@ def pre_commit():
     # Update examples/geojson/nodes.geojson and examples/geojson/spans.geojson
     subprocess.run(['libcoveofds', 'jsontogeojson', 'examples/json/network-package.json', 'examples/geojson/nodes.geojson', 'examples/geojson/spans.geojson'])
 
+    # Run mdformat
```
Comment (duncandewhurst) [inline manage.py:782]:
```suggestion
    # Format Markdown files
```

Comment (duncandewhurst) [reply manage.py:782]: Also, please add to the docstring for this function.

**Review by duncandewhurst — DISMISSED** [review]
Changes in this PR look fine, with one suggestion. However, I have some concerns about the way we're splitting https://github.com/Open-Telecoms-Data/open-fibre-data-standard/pull/166 into separate PRs: https://github.com/Open-Telecoms-Data/open-fibre-data-standard/pull/208/files#r1039027463

**Comment by ghost** [comment]
Just noticed that "docs/_static/docson/README.md" is caught by this - I think that's harmless, but do we want to try and exclude anything in docs/static? This is also something we could do later.

File: manage.py (hunk 3 above)
Comment (ghost) [reply manage.py:782]: Done by editing first commit.

File: docs/reference/schema.md (hunk 2 above)
Comment (ghost) [reply docs/reference/schema.md]: Fixed in second commit on this PR.

**Comment by ghost** [comment]
> However, I have some concerns about the way we're splitting https://github.com/Open-Telecoms-Data/open-fibre-data-standard/pull/166 into separate PRs

Will discuss #166 on that PR so that we can keep this PR on track.

**Review by lgs85 — APPROVED** [review]

**Pull request merged**